File: 7KYU/Count consonants/solution.c
#include <stddef.h>
#include <string.h>

size_t consonant_count(const char *string) {

    int l = strlen(string);
    size_t cnt = 0;
  
    for(int i = 0; i < l; i++) {
      char c = string[i];
      
      if((c >= 'b' && c <= 'z' && c != 'e' && c != 'i' &&
         c != 'o' && c != 'u') ||
         (c >= 'B' && c <= 'Z' && c != 'E' && c != 'I' &&
         c != 'O' && c != 'U'))  {
        cnt++;
      }
    }
    return cnt;

}

File: 7KYU/Count consonants/solution.cs
using System;
using System.Collections.Generic;
public class Kata
{
  public static int ConsonantCount(string str)
  {
    var vowels = new HashSet<char>(){'a', 'e', 'i', 'o', 'u'};
    int cnt = 0;
    foreach(var c in str.ToLower())
    {
       if(!vowels.Contains(c) && char.IsLetter(c))
         cnt++;
    }
    return cnt;
  }
}